src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxIPGlobalProperties.cs

```diff
@@ -1,15 +1,19 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Runtime.Versioning;
+
 namespace System.Net.NetworkInformation
 {
     internal sealed class LinuxIPGlobalProperties : UnixIPGlobalProperties
     {
+        [UnsupportedOSPlatform("android")]
         public override TcpConnectionInformation[] GetActiveTcpConnections()
         {
             return StringParsingHelpers.ParseActiveTcpConnectionsFromFiles(NetworkFiles.Tcp4ConnectionsFile, NetworkFiles.Tcp6ConnectionsFile);
         }
 
+        [UnsupportedOSPlatform("android")]
```

Comment:
These are overrides on an internal type.  What does marking just these as unsupported help with?

Reply:
They throw PNSE on Android, at least in tests, e.g.:
```
<failure exception-type="System.PlatformNotSupportedException">
    <message><![CDATA[System.PlatformNotSupportedException : The information requested is unavailable on the current platform.]]></message>
    <stack-trace><![CDATA[   at System.Net.NetworkInformation.StringParsingHelpers.ParseActiveTcpConnectionsFromFiles(String tcp4ConnectionsFile, String tcp6ConnectionsFile)
        at System.Net.NetworkInformation.LinuxIPGlobalProperties.GetActiveTcpConnections()
        at System.Net.NetworkInformation.Tests.IPGlobalPropertiesTest.IPGlobalProperties_TcpActiveConnections_Succeed(IPAddress address) in /Users/maxim/dev/runtime/src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/IPGlobalPropertiesTest.cs:line 92
    --- End of stack trace from previous location ---]]></stack-trace>
</failure>
```

Reply:
Sure. But in general these attributes are to help an analyzer flag problematic use.  What are these specific applications of the attribute going to flag? What code now produces a warning that didn't previously?

Reply:
You are right, no any warnings appear during the library build and a quick search shows that there is no applications of marked methods except in related tests. Should I revert my change in this case?

Reply:
If it's that IPGlobalProperties.GetActiveTcpConnections doesn't work on android, then presumably it should be the base public method on the public type that needs annotating. (It would also need to be in the ref, not just the src.)

@buyaa-n, does the analyzer not flag cases like this where an override is saying it doesn't support a platform but the base makes no such claim?

Reply:
> @buyaa-n, does the analyzer not flag cases like this where an override is saying it doesn't support a platform but the base makes no such claim?

No, it doesn't flag/support anything related to overload/inheritance, we planned to flag implicit constructor invocations in derived type https://github.com/dotnet/roslyn-analyzers/issues/4404 only. If we decide to add this, it could be added to the platform attributes correctness analyzer